save-cloud-common/src/commonMain/kotlin/org/cqfn/save/test/TestDto.kt

```diff
@@ -11,13 +11,15 @@ import kotlinx.serialization.Serializable
  * @property hash hash of file content
  * @property testSuiteId id of test suite, which this test belongs to
  * @property pluginName name of a plugin which this test belongs to
+ * @property tags list of tags of current test
  */
 @Serializable
 data class TestDto(
     val filePath: String,
     val pluginName: String,
     val testSuiteId: Long,
     val hash: String,
+    val tags: String?,
```

Comment:
Maybe, we can store tags as lists in DTO and split when in `toDto` methods in entities?